frontend/front/src/features/breadcrumb/BreadcrumbNav.js

```diff
@@ -0,0 +1,47 @@
+import { Breadcrumbs, Link, Stack, Typography } from "@mui/material";
+import { useDispatch, useSelector } from "react-redux";
+import { Link as RouterLink } from "react-router-dom";
+import { selectBreadcrumbs, setBreadcrumbs } from "./breadcrumbSlice";
+
+export const BreadcrumbNav = () => {
+  const dispatch = useDispatch();
+
+  // Make shallow copy of links to modify here
+  const links = [...useSelector(selectBreadcrumbs)];
+
+  // Clicking a previous breadcrumb pops links in list until matching the previous breadcrumb
+  const handleClick = (url) => {
+    while (links.at(-1)?.url !== url && links.length) {
+      links.pop();
+    }
+    dispatch(setBreadcrumbs(links));
+  };
+
+  return (
+    <Stack justifyContent="center" alignItems="center" width="100%" mt="10px">
```

Comment:
I feel like breadcrumbs are usually justified to the left? It feels weird to me that they're centered for whatever reason.